assets/config/config.go

```diff
@@ -0,0 +1,46 @@
+package configassests
+
+import (
+	_ "embed"
+
+	"github.com/scylladb/scylla-operator/pkg/helpers"
+	"sigs.k8s.io/yaml"
+)
+
+var (
+	//go:embed "config.yaml"
+	projectConfigString string
+	Project             = helpers.Must(func() (*ProjectConfig, error) {
+		c := &ProjectConfig{}
+		err := yaml.UnmarshalStrict([]byte(projectConfigString), c)
+		return c, err
+	}())
+)
+
+type OperatorConfig struct {
+	ScyllaDBVersion string `json:"scyllaDBVersion"`
+	// scyllaDBEnterpriseVersionNeedingConsistentClusterManagementOverride sets enterprise version
+	// that requires consistent_cluster_management workaround for restore.
+	// In the future, enterprise versions should be run as a different config instance in its own run.
+	ScyllaDBEnterpriseVersionNeedingConsistentClusterManagementOverride string `json:"scyllaDBEnterpriseVersionNeedingConsistentClusterManagementOverride"`
```

Comment:
I would change this to `scyllaDBVersionNeedingConsistentClusterManagementOverride` - this way we'll be able to test this with 5.4.Z for OS suite and 2024.1.Z for enterprise suite

Reply:
It uses enterprise repo
https://github.com/scylladb/scylla-operator/blob/f27f72f4273a6ef2e60b460b66987c8a5e59dae1/test/e2e/set/scyllacluster/scyllamanager_object_storage.go#L458

Reply:
We could configure both. It can be done once we actually have a separate suite for enterprise though.